pkg/operator/encryption/controllers/key_controller.go

```diff
@@ -194,7 +194,14 @@ func (c *keyController) checkAndCreateKeys() error {
 	if errors.IsAlreadyExists(createErr) {
 		return c.validateExistingSecret(keySecret, newKeyID)
 	}
-	return createErr
+	if createErr != nil {
+		c.eventRecorder.Warningf("EncryptionKeyCreateFailed", "Secret %q failed to create: %v", keySecret.Name, err)
```

Comment:
should we really create these events? They show up as degraded anyway.

Reply:
@sttts why not, at least we will see when this started to fail and which secret

Reply:
if we go degraded, we see the error message too, don't we?